code/modules/clothing/under/jobs/civilian.dm

```diff
@@ -25,8 +25,8 @@
 	item_state = "b_suit"
 
 /obj/item/clothing/under/rank/cargotech
-	name = "cargo worker's jumpsuit"
-	desc = "A pair of jeans and turtleneck worn by the cargo workers."
+	name = "guild worker's jumpsuit"
+	desc = "A pair of jeans and turtleneck worn by the guild workers."
```

Comment:
```suggestion
	desc = "Pair of jeans and turtleneck worn by the guild workers."
```
See previous comments. May or may not be a good idea, take with a grain of salt.